terms/evaluationmethods_ae.tex

```diff
@@ -0,0 +1,131 @@
+% , in the spirit of the Turing test,
+involves training Generative Adversarial Networks\index{generative adversarial networks}~\citep{Goodfellow2014} to distinguish between human and machine-generated outputs.
+
+Another way to overcome the lack of metrics and data is to shift the task on which a model is evaluated, i.e.\ conduct \emph{extrinsic}\index{evaluation methods!evaluation!extrinsic} evaluation~\citep{Belz2008} that commonly involves assessment of a model's performance in a ``downstream'' application rather than the training task, as opposed to \emph{intrinsic}\index{evaluation methods!evaluation!intrinsic} evaluation.
+For instance, if the goal is to evaluate a feature reduction mechanism, one can measure its effect on the accuracy of sentiment classification~\citep{Schnabel2015}.
+%
+An example of manual extrinsic evaluation is A/B studies\index{evaluation methods!A/B testing}~\citep{Hofmann2016} aimed to quantitatively measure a change in user experience before and after deploying a new machine learning model.
+A/B testing consists of dividing users into two groups A and B and comparing them against each other according to some metrics.
+%
+A/B tests can be employed to investigate the average time it takes users to find the desired webpage depending on the search algorithm~\citep{Manning2008}
+% or how quickly users can book a flight using different conversational agents~\citep{Jurafsky2009};
+or the change in time users spend on a website after introducing a new recommendation system~\citep{Davidson2010}.
+
+\section{Statistical Significance Testing}\index{evaluation methods!statistical significance testing|(}
+
+Whenever a new algorithm is proposed, it needs to be compared against either a random baseline or previously published results for the task and dataset at hand.
+However, often improvement is rather incremental~\citep{Hand2006}, so it might not be evident if it is of any importance.
+To decide whether a change is significant, \emph{statistical significance tests} are encouraged, at least in case of scarce testing data~\citep{Drummond2010}.
+
+Details about general statistical tests can be found in~\citep{Urdan2011}, whereas~\citet{Japkowic2011} provide recommendations on the correct use of such tests for classifier comparison.
+In evaluating machine learning techniques, one needs to be cautious when using standard parametric\index{evaluation methods!statistical significance testing!parametric} tests such as the paired $t$-test~\citep{Urdan2011} because of the assumptions expected to be met, e.g.\ equal variance of obtained metrics~\citep{Japkowic2011}. In contrast, \emph{non-parametric}\index{evaluation methods!statistical significance testing!non-parametric} tests do not impose any restriction on the population distribution which makes them more applicable, although less powerful~\citep{Colquhoun1971}.
+%
+Below is an illustration of applying a non-parametric test.
```

Comment:
Refer to section (next section)  not using propositions "above" and "below" (these may move in the document)

Reply:
Done. Replaced with "what follows is", as I do not see much problem with this formulation even if the next section moves to the next page.